src/models/converters/transactions/mod.rs

```diff
@@ -150,9 +167,11 @@ impl MultisigTransaction {
         }
     }
 
-    fn to_ether_transfer(&self) -> Transfer {
+    fn to_ether_transfer(&self, info_provider: &mut dyn InfoProvider) -> Transfer {
         Transfer {
             sender: self.safe.to_owned(),
+            sender_info: None,
```

Comment:
Little very unimportant details but it triggers my ocd: can we have `sender`, `sender_info`, `recipient` and `recipient_info` always defined in the same order (currently the token transfers have a different order than the ether transfer) 😄